### PR TITLE
Update grouppolicy.md

### PR DESCRIPTION
**Pull request by crutkas**
Moving the PT https://github.com/microsoft/PowerToys/blob/main/doc/gpo/README.md over here and pointing the PT doc over

### CODE REVIEW

File: hub/powertoys/grouppolicy.md [hunk 1]
```diff
@@ -1,43 +1,124 @@
...
+### Installer and Updates
+
+#### Disable per-user installation
+
+This policy configures whether PowerToys per-user installation is allowed or not.
+
+- If enabled, per-user installation is not allowed.
+- If disabled or not configured, per-user installation is allowed.
+
+ **Note:** You can set this policy only as Computer policy.
```
Comment (alvinashcraft) [inline hub/powertoys/grouppolicy.md]: The content on the page all looks good. Do you want to use the MS Learn Note style that stands out for the notes on this page?
```suggestion
> [!NOTE]
> You can set this policy only as Computer policy.
```

**Comment by crutkas** [comment]
@alvinashcraft done

**Review by alvinashcraft — APPROVED** [review]
Looks good!

**Comment by crutkas** [comment]
once this is done, i'll merge the PT one as well.  Alvin, can you do me a solid and validate https://github.com/microsoft/PowerToys/pull/29190 is solid?

**Comment by alvinashcraft** [comment]
> once this is done, i'll merge the PT one as well. Alvin, can you do me a solid and validate [microsoft/PowerToys#29190](https://github.com/microsoft/PowerToys/pull/29190) is solid?

Sure thing. I'll go take a look.

File: hub/powertoys/grouppolicy.md [hunk 2]
```diff
@@ -1,43 +1,128 @@
 ---
 title: PowerToys Group Policy
 description: Group policy objects
```
Comment (htcfreek) [inline hub/powertoys/grouppolicy.md:3]: Wrong description. Please change it to `Group policy documentation` or something similar.

File: hub/powertoys/grouppolicy.md [hunk 3]
```diff
@@ -1,43 +1,128 @@
...
+4. If your domain has more than one domain controller, the new ADMX files will be replicated to them at the next domain replication interval.
 
 ### Scope
 
 You will find the policies under "Administrative Templates/Microsoft PowerToys" in both the Computer Configuration and User Configuration folders. If both settings are configured, the setting in Computer Configuration takes precedence over the setting in User Configuration.
 
 ## Policies
 
-Policy can force a PowerToys utility to be enabled or disabled for users covered by the policy.
+### Configure global utility enabled state
```
Comment (htcfreek) [inline hub/powertoys/grouppolicy.md:33]: Policy is supported on PT 0.75.0 and newer.

File: hub/powertoys/grouppolicy.md [hunk 4]
```diff
@@ -1,43 +1,128 @@
...
+
+This policy allows you to disable automatic update checks running in the background. (The manual check in PT Settings is not affected by this policy.)
+
+- If enabled, the automatic update checks are disabled.
+- If disabled or not configured, the automatic update checks are enabled.
+-->
+
+### PowerToys Run
+
+#### Configure enabled state for all plugins
```
Comment (htcfreek) [inline hub/powertoys/grouppolicy.md:101]: Policy is supported on PT 0.75.0 and newer.

File: hub/powertoys/grouppolicy.md [hunk 5]
```diff
@@ -1,43 +1,128 @@
...
+- If you enable this setting, the plugins will be always enabled and the user won't be able to disable it.
+- If you disable this setting, the plugins will be always disabled and the user won't be able to enable it.
+- If you don't configure this setting, users are able to disable or enable the plugins.
+
+You can override this policy for individual plugins using the policy "Configure enabled state for individual plugins".
+
+> [!NOTE]
+> Changes require a restart of PowerToys Run.
+
+#### Configure enabled state for individual plugins
```
Comment (htcfreek) [inline hub/powertoys/grouppolicy.md:114]: Policy supported on PT 0.75.0 and newer.

**Review by htcfreek — COMMENTED** [review]
There is a small nit that needs to be fixed later.

And you merged to early! Some Group Policies coming with the 0.75.0 release. (Can you please add a note for the marked policies?)

**Comment by alvinashcraft** [comment]
> There is a small nit that needs to be fixed later.
> 
> And you merged to early! Some Group Policies coming with the 0.75.0 release. (Can you please add a note for the marked policies?)

Oh man. I'm sorry. I'll go create a PR with these changes and add you to the review. Thanks for the heads-up, @htcfreek

**Comment by alvinashcraft** [comment]
I just merged your suggested changes via the windows-dev-docs-pr repo. It should be live shortly. Thanks again!

**Pull request merged**